docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,83 @@
+
+## Table
+
+| Name of feature            | Dependabot                                                                                                                                                                   | Renovate                                                                                                 |
+| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Dependency Dashboard       | No                                                                                                                                                                           | Yes                                                                                                      |
+| Grouped updates            | Yes, create [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) manually     | Yes, use community-provided groups, or create your own                                                   |
+| Monorepo support           | Unknown                                                                                                                                                                      | Yes                                                                                                      |
+| Supported platforms        | GitHub only                                                                                                                                                                  | GitHub, GitLab, Bitbucket, Azure, Gitea, see [full list](https://docs.renovatebot.com/modules/platform/) |
+| Supported languages        | [List for Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems) | [List for Renovate](https://docs.renovatebot.com/modules/manager/)                                       |
+| Customizability            | Low                                                                                                                                                                          | High                                                                                                     |
+| Show changelogs            | Yes                                                                                                                                                                          | Yes                                                                                                      |
```

Comment:
Besides change logs, commit history and release notes are included  with Dependabot.

Reply:
I first need hands-on experience with Dependabot to make a meaningful comparison. See issue:

- https://github.com/renovatebot/renovate/issues/24689